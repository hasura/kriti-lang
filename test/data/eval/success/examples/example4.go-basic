{
  "person": {
    "name": {{$[0].name}},
    "age": {{$[0].age}},
    "friends": [
      "hello",
{{ range _, x := $[0].friends }}
      {
        "id": {{x.id}},
        "name": {{x.name}}
      }
{{ end }},
      "world"
    ]
  }
}