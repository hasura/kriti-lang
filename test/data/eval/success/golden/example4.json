{
    "person": {
        "age": 26,
        "friends": [
            "hello",
            [
                {
                    "id": 0,
                    "name": "Wilkerson Gill"
                },
                {
                    "id": 1,
                    "name": "Gonzalez Long"
                },
                {
                    "id": 2,
                    "name": "Hanson Ellison"
                }
            ],
            "world"
        ],
        "name": "Shepard Mccall"
    }
}